projects/ngx-d3timeline/src/lib/drag-utils.ts

```diff
@@ -45,41 +15,36 @@ export function getCurrentlyDraggedActivity(
   );
 }
 
-function getDropActivitySeries(
-  bandScale: BandScale,
-  dragEvent: TimelineDragEvent,
-  timeOrientation: Orientation
-) {
-  const invert = getInverseBandScale(bandScale);
-  return dragEvent && timeOrientation === Orientation.Vertical
-    ? invert(dragEvent.x)
-    : invert(dragEvent.y);
-}
-
-function shiftedTimesForDraggingActivity(
+export function shiftUpdatedTimes(
```

Comment:
as above this method may change, if it stays as is rename it something like `setUpdatedTimes` or `setUpdatedTimesForActivity`.

Is it is not shifting anything as current name implies.

Reply:
done